tests/core/gui/devices_widget/test_greet.py

```diff
@@ -263,6 +264,7 @@ async def test_greet_device(GreetDeviceTestBed):
     await GreetDeviceTestBed().run()
 
 
+@pytest.mark.skipif(OXIDIZED, reason="greet_device_widget is still visible")
```

Comment:
do we really have a good reason to disable this test ?

Reply:
will be removed given we only enable basic tests for the unstable oxidized client connection feature